docs/modules/ROOT/pages/dataformats.adoc

```diff
@@ -550,4 +550,36 @@ The following write parameters are supported:
 
 |===
 
-IMPORTANT: It is not recommended to call `DS.Formats.write(...)` directly in the DataSonnet code, because it will produce a String representation of a POJO, i.e. `pojo.toString()`. Normally POJO outputs are expected to be the end results of the transformations and are controlled by the appropriate xref:headers.adoc[headers] at the beginning of the transformation code.
\ No newline at end of file
+IMPORTANT: It is not recommended to call `DS.Formats.write(...)` directly in the DataSonnet code, because it will produce a String representation of a POJO, i.e. `pojo.toString()`. Normally POJO outputs are expected to be the end results of the transformations and are controlled by the appropriate xref:headers.adoc[headers] at the beginning of the transformation code.
```

Comment:
Oh, I missed this. It is entirely disallowed now, programmatically, so this section can be removed

Reply:
Done